### DISCUSSION TITLE
Free domain namecheap

### DISCUSSION THREAD

**Discussion by essalihi** [community/community#20336] [GitHub Education] [answered]
Hello,
I am having a problem to get my free domain in namecheap offered by the github student pack
idk how i can solve this problem
i would realy appreciate any help 
thank you

**Comment by D7EAD** [comment, marked as answer]
Hi!

If you're having trouble acquiring the free domain name from Namecheap, I'd recommend looking [here](https://nc.me/). If you're having trouble understanding how to acquire this free domain name **via** the Github Student Pack, I'd recommend contacting Github Education directly [here](https://support.github.com/contact/education).

**Reply by essalihi** [reply]
I solved the problem i was just submiting my school email 
while i should use my personal email (which is my primary email in github)
thank you so much @D7EAD
